src/V3Broken.cpp

```diff
@@ -237,6 +237,7 @@ class BrokenCheckVisitor final : public VNVisitorConst {
         processAndIterate(nodep);
     }
     void visit(AstNodeModule* nodep) override {
+        if (nodep->dead()) return;
```

Comment:
Dead nodes cannot be exempt from having proper linkage, or we may have memory corruption or other problems hiding.